Backends/PostScript/font/standard-metrics.lisp

```diff
@@ -1,5 +1,6 @@
 
-(in-package :clim-postscript) 
+(in-package :clim-postscript-font)
+
```

Comment:
why do we have new package clim-postscript-font? because it is reused by pdf backend? If yes, shouldn't it be named `clim-document-font` or something, and put in `Backends/Standard`?